odo/backends/tests/tripsmall.csv

```diff
@@ -0,0 +1,31 @@
+4C005EEBAA7BF26B84B21586332488A2,351BE7D984BE17DB2FA80A748E816472,CMT,1,N,2013-01-07 22:09:59,2013-01-07 22:19:50,1,591,1.7,-73.973732,40.756287,-73.998413,40.756832,0.0,0.0,10.0,0.5,9.0,CSH,0.5
+7D99C30FCE69B1A9DD27E2AEAC9BFA0C,460C3F57DD9CB2265DB75B14CD70224D,CMT,1,N,2013-01-07 17:18:16,2013-01-07 17:20:55,1,158,0.7,-73.968925,40.767703999999995,-73.96199,40.776565999999995,0.0,0.0,6.0,0.5,4.5,CSH,1.0
+E6FBF80668FE0611AEA44FD9574A7E32,36773E80775F26CD1158EB5450A61C79,CMT,1,N,2013-01-07 06:08:51,2013-01-07 06:13:14,1,262,1.7,-73.96212,40.769737,-73.979561,40.755390000000006,0.0,0.0,7.5,0.5,7.0,CSH,0.0
+0C5296F3C8B16E702F8F2E06F5106552,D2363240A9295EF570FC6069BC4F4C92,CMT,1,N,2013-01-07 22:25:46,2013-01-07 22:36:56,1,669,2.3,-73.989708,40.756714,-73.977615,40.787575,0.0,0.0,11.5,0.5,10.5,CSH,0.5
+D8AADA4E722C87C10E609654612630DD,8F1E4F779A4D67431C3FCFA4E70A87DC,CMT,2,N,2013-01-10 23:41:51,2013-01-11 00:09:11,1,1640,17.5,-73.78331,40.648765999999995,-73.98891400000001,40.748207,4.8,0.0,57.3,0.5,52.0,CSH,0.0
+B6FC8357E0D53B0F1FA897D536A20F5C,113A5BA8513934DEE97A342E3535DE96,CMT,1,N,2013-01-07 18:05:36,2013-01-07 18:23:50,4,1094,3.8,-73.956505,40.771278,-73.996368,40.732459999999996,0.0,0.0,17.0,0.5,15.5,CSH,1.0
+E12F6AF991172EAC3553144A0AF75A19,06918214E951FA0003D1CC54955C2AB0,CMT,1,N,2013-01-08 13:29:25,2013-01-08 13:37:52,1,507,1.2,-73.95578,40.77932,-73.967285,40.763344000000004,0.0,0.0,8.0,0.5,7.5,CSH,0.0
+4C005EEBAA7BF26B84B21586332488A2,351BE7D984BE17DB2FA80A748E816472,CMT,1,N,2013-01-07 21:13:02,2013-01-07 21:22:31,1,568,1.1,-73.978439,40.764679,-73.977684,40.777004,0.0,0.0,9.0,0.5,8.0,CSH,0.5
+E9BA494DC1A1F26D186394EFBB8327CF,BBF604B3A2426BFBAAC982AAAA0BF1B5,CMT,1,N,2013-01-01 18:36:53,2013-01-01 18:39:04,1,131,0.5,-73.992172,40.749953999999995,-73.99675,40.744553,0.0,0.0,4.5,0.5,4.0,CSH,0.0
+E6FBF80668FE0611AEA44FD9574A7E32,36773E80775F26CD1158EB5450A61C79,CMT,1,Y,2013-01-07 08:17:06,2013-01-07 08:22:46,1,340,1.5,-73.992111,40.689701,-74.00715600000001,40.679295,0.0,0.0,7.5,0.5,7.0,CSH,0.0
+0C5296F3C8B16E702F8F2E06F5106552,D2363240A9295EF570FC6069BC4F4C92,CMT,1,N,2013-01-07 22:39:03,2013-01-07 22:44:16,1,312,1.5,-73.97865300000001,40.787735,-73.96579,40.805321,0.0,0.0,8.0,0.5,7.0,CSH,0.5
+A3B17384165197E18CA0A1BB61277EE9,B8396B62883EA332EFD2771A6B031D05,CMT,1,N,2013-01-07 06:26:32,2013-01-07 06:28:24,2,111,0.5,-73.965317,40.769375,-73.96713299999999,40.763699,0.0,0.0,5.0,0.5,4.0,CSH,0.5
+8E189DABE265CC03FEF4BFC695B32559,906939B31C0B93768242A58A682412B8,CMT,1,N,2013-01-01 14:02:01,2013-01-01 14:27:39,2,1537,10.2,-73.862709,40.769141999999995,-73.982079,40.762295,4.8,0.0,36.3,0.5,31.0,CSH,0.0
+312E0CB058D7FC1A6494EDB66D360CD2,7B5156F38990963332B33298C8BAE25E,CMT,1,N,2013-01-05 11:54:49,2013-01-05 12:03:48,1,539,0.8,-73.977127,40.74831,-73.990913,40.751053000000006,0.0,0.0,8.0,0.5,7.5,CSH,0.0
```

Comment:
Nice dataset to have around.  Might want to ensure that this gets packaged when users pip or conda install.